### PR TITLE
Refactor populateTable function

### PR DESCRIPTION
**Pull request by aliroemhildt**

### CODE REVIEW

**Review by cagallo — APPROVED** [review]
Looks great!

File: src/scripts.js [hunk 1]
```diff
@@ -85,32 +85,25 @@ addToPantryButton.addEventListener('click', addIngredientToPantry);
...
-    let response = await post(info);
-    // throw "error"
-  } catch (error) {
-    console.log(error);
-  }
+    let response = await post(info)
+    .catch(error => console.log(error))
+}
+
+async function getPantry() {
```
Comment (anewb87) [inline src/scripts.js:102]: Looks nice and clean!

**Review by kammon10 — APPROVED** [review]

File: src/scripts.js [hunk 2]
```diff
@@ -85,32 +85,25 @@ addToPantryButton.addEventListener('click', addIngredientToPantry);
...
+  let response = await fetch("http://localhost:3001/api/v1/users")
+  .then(response => response.json())
+  .then(data => reassignUserPantry(data))
+  .catch(err0r => console.log(error))
```
Comment (anewb87) [inline src/scripts.js:106]: Got a whoops on this line that we're just about to fix.

File: src/scripts.js [hunk 3]
```diff
@@ -455,6 +425,25 @@ function populatePantry() {
   })
 }
 
+function createTable() {
```
Comment (anewb87) [inline src/scripts.js:428]: Trying to get our code more single-purpose.

**Review by anewb87 — APPROVED** [review]
Like the refactoring.

**Pull request merged**